converter/internal/prometheusconvert/testdata/discovery.river

```diff
@@ -0,0 +1,69 @@
+prometheus.remote_write "default" {
```

Comment:
It would be nice if we had a little more control over the order these converted in, so that it's roughly:

1. Discovery component(s) 
2. Discovery relabel component(s) (if any)
3. Prometheus scrape component(s) 
4. Prometheus relabel component(s) (if any) 
5. Prometheus remote_write

If it's too much effort, it's probably not worth it right now.

Reply:
This has been implemented a couple ways in the past 2 commits. The most recent 1 will organize the blocks into categories so they don't need to be sorted at the end and get appended to the file as we want them.